docs/topics/proc_deploying-the-health-check-booster-using-the-oc-cli-client.adoc

```diff
@@ -60,10 +61,10 @@ ifdef::built-for-nodejs[]
 $ npm install && npm run openshift
 ----
 +
-These commands install any missing module dependencies, then using the xref:about-nodeshift[Nodeshift] module, deploy the booster on OpenShift.
+These commands install any missing module dependencies, then using the xref:about-nodeshift[Nodeshift] module, deploy the example on OpenShift.
 endif::built-for-nodejs[]
 
-. Check the status of your booster and ensure your pod is running.
+. Check the status of your example and ensure your pod is running.
```

Comment:
Check the status of your example--> Check the status of your example application